wp-content/themes/core/theme.json

```diff
@@ -1,10 +1,77 @@
 {
-	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
-	"settings": {
-		"layout": {
-			"contentSize": "80ch",
-			"wideSize": "1200px"
-		}
-	}
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "color": {
+      "custom": false,
+      "customDuotone": false,
+      "customGradient": false,
+      "defaultGradients": false,
+      "defaultPalette": false,
+      "duotone": [],
+      "gradients": [],
+      "link": false,
+      "palette": [
```

Comment:
Note: I didn't define all colors from the styleguide here as I think we'll want a curated list based on what should be available for content editors. I assume as we get further into core block styles, color theming, etc we'll be able to hone these in.

Reply:
I say we define them all here. We can tailor a subset of colors available for different blocks/contexts, but I'd prefer to have everything available from the json file initially.

Additionally, I think this global palette should be named 1:1 with what is defined in [the global palette in Figma](https://www.figma.com/file/zbjC33tZJz5A901jeXM7zv/Website-Design-System?node-id=804-5786&t=5pO9WEjMQHEbgwrv-0) (same naming even). Once we define contextual palettes for different blocks/use cases, we can add the relevant names (foreground, text, etc).

Reply:
Makes sense, I like that, will update 👍

Reply:
✔️ These have been updated